firebase/functions/src/util.ts

```diff
@@ -46,32 +27,24 @@ export const getUser = async function(username: string): Promise<Firestore.Docum
  */
 export const getDeck = async function(username: string, deckId: string): Promise<Firestore.DocumentSnapshot> {
     const userDoc = await getUser(username);
-    let deckDoc;
-    try {
-        deckDoc = await userDoc.ref.collection('Decks').doc(deckId).get();
-    } catch (e) {
-        console.error('An error occured reading from Firestore', e);
-        throw createError(500);
-    }
+    const deckDoc = await userDoc.ref.collection('Decks').doc(deckId).get();
```

Comment:
You could use the other format here to simplify a little bit, like the SDK does (`Decks/deckId`).

Reply:
(There are supposed to be backticks around that)

Reply:
```
`Decks/${deckId}`
```

Reply:
It doesn't seem like `DocumentReference` exposes a `doc` method that I can use like that. https://firebase.google.com/docs/reference/js/firebase.firestore.DocumentReference